src/Accounts/Accounts/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 
 ## Upcoming Release
+* Enable WAM as default way of interactive login flow.
```

Comment:
We might want to be more informative about WAM being default (for example, add how to switch back to the legacy experience) and less informative about package upgrade (because of the length limitation of psd1)